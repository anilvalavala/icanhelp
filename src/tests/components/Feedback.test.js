import React from 'react';
import { shallow } from 'enzyme';
import Feedback from '../../components/Feedback';

test('Test Feedback component', () => {
    const wrapper = shallow(<Feedback />);
    expect(wrapper).toMatchSnapshot();
});
