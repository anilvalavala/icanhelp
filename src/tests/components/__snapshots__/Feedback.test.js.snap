// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Feedback component 1`] = `
<div>
  <h1>
    Feedback
  </h1>
</div>
`;
